packages/shared/src/components/post/analytics/BoostingLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {
+  Typography,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+
+export const BoostingLabel = () => (
+  <Typography
+    tag={TypographyTag.Span}
+    type={TypographyType.Footnote}
+    className="rounded-6 bg-action-comment-float px-1 py-px text-action-comment-default"
```

Comment:
We don't support this color out of box?
If it's applied in more places we can maybe adds it as typography color.